src/fides/api/models/privacy_experience.py

```diff
@@ -273,8 +279,9 @@ def get_related_privacy_notices(
         """
         if self.component == ComponentType.tcf_overlay:
             return []
+
         privacy_notice_query = get_privacy_notices_by_region_and_component(
-            db, self.region, self.component  # type: ignore[arg-type]
+            db, [self.region.value, self.region_country], self.component  # type: ignore[arg-type]
```

Comment:
NOTE: we should be careful here, this is a questionable change - need to review further...